src/puppetlabs/trapperkeeper/authorization/rules.clj

```diff
@@ -119,11 +123,20 @@
   [a b]
   (or (= a b) (some #{:any} [a b])))
 
+(defn- query-params-match?
```

Comment:
I think the rationale for not using `defn-` was that it makes it harder to write unit tests against those functions when your testing namespace is outside of the code's namespace.  To avoid that, we've instead frequently decorated functions intended to be "private" with a comment pragma like this:

https://github.com/puppetlabs/trapperkeeper/blob/trapperkeeper-1.1.0/src/puppetlabs/trapperkeeper/config.clj#L40-L53

I don't think we've always been consistent about that, though.  For example, we still have a number of `defn-` references in the trapperkeeper project and other PL Clojure projects like clj-http-client.

@KevinCorcoran - do you know what our "state of the art" practice for this is considered to be?